packages/features/bookings/lib/getBookingResponsesSchema.ts

```
@@ -103,7 +108,9 @@ function preprocess<T extends z.ZodType>({
           };
           try {
             parsedValue = JSON.parse(value);
-          } catch (e) {}
+          } catch {
+            // If parsing fails, we don't add the field to newResponses
```

Why do we have an empty catch block ?

I think it is a bad practice to have empty catch blocks.